### PR TITLE
Fix SHADERC_SKIP_TESTS configuration to actually skip all tests.

### PR DESCRIPTION
**Pull request by antiagainst**
Together with https://github.com/KhronosGroup/SPIRV-Tools/pull/301.

### CODE REVIEW

File: cmake/utils.cmake [hunk 1]
```diff
@@ -21,10 +21,10 @@ function(shaderc_default_c_compile_options TARGET)
     endif()
     if (NOT SHADERC_ENABLE_SHARED_CRT)
       if (WIN32)
-	# For MinGW cross compile, statically link to the libgcc runtime.
-	# But it still depends on MSVCRT.dll.
-	set_target_properties(${TARGET} PROPERTIES
-		LINK_FLAGS "-static -static-libgcc")
+  # For MinGW cross compile, statically link to the libgcc runtime.
```
Comment (dneto0) [inline cmake/utils.cmake]: Indentation was damaged here..  ?!

File: cmake/utils.cmake [hunk 2]
```diff
@@ -40,10 +40,10 @@ function(shaderc_default_compile_options TARGET)
     target_compile_options(${TARGET} PRIVATE -std=c++11)
     if (NOT SHADERC_ENABLE_SHARED_CRT)
       if (WIN32)
-	# For MinGW cross compile, statically link to the C++ runtime.
-	# But it still depends on MSVCRT.dll.
-	set_target_properties(${TARGET} PROPERTIES
-		LINK_FLAGS "-static -static-libgcc -static-libstdc++")
+  # For MinGW cross compile, statically link to the C++ runtime.
```
Comment (dneto0) [inline cmake/utils.cmake]: Simlar

**Comment by dneto0** [comment]
Ok except for bad changes to indentation

-1

File: cmake/utils.cmake (hunk 1 above)
Comment (antiagainst) [inline cmake/utils.cmake]: Hmm. I was to fix the mixed indent characters earlier. It seems just didn't put enough spaces for tabs.

File: cmake/utils.cmake (hunk 2 above)
Comment (antiagainst) [inline cmake/utils.cmake]: Done.

**Pull request merged**